src/components/Dashboard.svelte

```diff
@@ -1,41 +1,22 @@
 <script lang="ts">
-  import Connect from '$components/Connect.svelte';
   import Capacity from '$components/Capacity.svelte';
   import Provider from '$components/Provider.svelte';
-  import KeySelection from '$components/KeySelection.svelte';
-  import ProviderActions from '$components/ProviderActions.svelte';
   import DashboardHeader from '$components/DashboardHeader.svelte';
   import ChainStatus from '$components/ChainStatus.svelte';
   import type { ChainInfo } from '$lib/storeTypes';
 
-  import {
-    storeBlockNumber,
-    storeChainInfo,
-    storeConnected,
-    storeMsaInfo,
-    storeToken,
-    transactionSigningAddress,
-  } from '$lib/stores';
-  import { storeValidAccounts } from '$lib/stores/accountsStore';
-
-  const onChangeTxnSigningAddress = (evt: Event) => {
-    let option = evt.target as HTMLOptionElement;
-    storeMsaInfo.set({ isProvider: false, msaId: 0, providerName: '' });
-    transactionSigningAddress.set(option.value);
-  };
+  import { storeBlockNumber, storeChainInfo, storeConnected, storeToken } from '$lib/stores';
 
   let token = '';
 
   storeToken.subscribe((val) => (token = val));
   let blockNumber = 0n;
   let epochNumber = 0n;
   let connected = false;
```

Comment:
```suggestion
  let blockNumber = $storeBlockNumber;
  let epochNumber = $storeChainInfo.epochNumber;
  let connected = $storeConnected;
```

I think we can just do this and delete lines 18-20, or refer to the values using the $___ directly.